src/routes/privacy/index.tsx

```diff
@@ -291,7 +287,11 @@ export default component$(() => {
         laws. Our international transfers of Personal Data are based on the EU
         Commission&apos;s standard contractual clauses.
       </Text>
-
+      <Text>
+        Currently, we are not processing data outside the EU/EES and the data is
```

Comment:
"Currently" doesn't sound very convincing. Let's go with something like: "We do not store, or process data outside of.. All of Our data is stored in Stockholm, Sweden." 

Do we want to include the part about offering storage in different countries right now? I feel like we could add that part when we get a use case for it, and have the infrastructure to back it up.